include/tattoowidget.hpp

```diff
@@ -18,29 +18,27 @@
 
 #include <QWidget>
 
-class TattooWidget : public QWidget
-{
-	Q_OBJECT
+class TattooWidget : public QWidget {
+  Q_OBJECT
 public:
-	TattooWidget(QWidget *p);
-	~TattooWidget();
-	void setRadius(int,int);
-	void setImage(QImage&);
+  TattooWidget(QWidget *p);
```

Comment:
Codacy found an issue: [Class 'TattooWidget' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)